tests/lifecycle/textanalytics_cases_test.go

```diff
@@ -0,0 +1,60 @@
+// +build !unit
+
+package lifecycle
+
+var textanalyticsTestCases = []serviceLifecycleTestCase{
+	{ // Text analytics free tier
+		group:     "textanalytics",
+		name:      "text-analytics-free",
+		serviceID: "8f6c848a-4ce1-4a69-9248-63545d3e7e9c",
+		planID:    "d5a0f91f-10da-42fc-b792-656a616d9ec2",
+		provisioningParameters: map[string]interface{}{
+			"location": "eastus",
+		},
```

Comment:
Could you also add "testCredentials" in the lifecycle cases? It is like an example how the credentials can be used. You can refer to MySQL: https://github.com/Azure/open-service-broker-azure/blob/master/tests/lifecycle/mysql_cases_test.go#L43.

Reply:
@zhongyi-zhang - I've added a test to validate a response code of 200 from the Text Analytics endpoint using the returned endpoint address and access key. This should be sufficient.

https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics.V2.0/operations/56f30ceeeda5650db055a3c7

Reply:
@neilpeterson cool, thanks for adding it!